Content/posts/launching-swift-package-collections copy.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2021-10-18 12:00
+title: Package Dependencies - Step One
+description:
+---
+
+We’ve said several times on this blog that one of the primary goals of the [Swift Package Index](https://swiftpackageindex.com) is to help you make better decisions about the dependencies you include in your Swift projects.
+
+In pursuit of that goal, we just launched a new feature that shows you how many dependencies each of the packages in our index has. 🚀
+
+Even better, it doesn’t just give you the number of top-level dependencies that a package makes in its `Package.swift` file. Instead, we inspect the `Package.resolved` file after the [build system](/posts/launching-language-and-platform-package-compatibility) has done the hard work of dependency resolution.
```

Comment:
maybe: `that a package declares in its …`

Reply:
Fixed! Thanks